app/controllers/api/v1/patients_controller.rb

```diff
@@ -225,6 +225,12 @@ def visit
     render json: service.fetch_full_visit(patient, program, params[:date])
   end
 
+  def patient_details_by_id
+    patient_id = params[:patient_id]
```

Comment:
I meant here: can you use "api/v1/patient/<patient_id>"

Reply:
Am working on it